.pre-commit-config.yaml

```diff
@@ -16,10 +17,18 @@ repos:
       - id: requirements-txt-fixer
       - id: check-added-large-files
       - id: check-merge-conflict
-  - repo: https://github.com/pre-commit/mirrors-pylint
-    rev: 'v3.0.0a5'  # Use the sha / tag you want to point at
+  - repo: local
```

Comment:
ooooh :( :(

What's wrong with merely updating this to a new sha?

I don't have pylint or most other dev tools installed globally since there's too many different versions available between different repos.

Reply:
> I don't have pylint or most other dev tools installed globally

that's a good point...didn't think about it. I'll roll it back to remote repo. One can do local `pylint` locally if they need more thorough check